prettier-settings.md

```diff
@@ -0,0 +1,25 @@
+# **Prettier Settings**
```

Comment:
I think we are not understanding each other.  You have only completed one of the ACs - you've formatted everything:
![image](https://user-images.githubusercontent.com/29131689/77262398-c6675400-6c52-11ea-947b-20ec4439c6f7.png)

For the other two, you need to *install* prettier into our project - that requires a config file or an eslintrc or something to that effect. Also, the issue requests a pre-commit hook for automatic lint checking, which I would expect to see in the package.json.